doc/rippled-example.cfg

```diff
@@ -374,12 +374,6 @@
 #       When set, activates the autoconnect feature. This maintains outgoing
 #       connections using PeerFinder's "Outgoing Connection Strategy."
 #
-#   http_handshake = 0 | 1
-#
-#       When set, outgoing peer connections will handshaking using a HTTP
-#       request instead of the legacy TMHello protocol buffers message.
-#       Incoming peer connections have their handshakes detected automatically.
```

Comment:
I answered my own question: Set `http_handshake = 1` on my dogfood rippled, started it up, and it had 15 outbound peer connections within moments.